packages/bigquery/src/table.js

```diff
@@ -1015,18 +1015,19 @@ Table.prototype.import = function(source, metadata, callback) {
  * //-
  * function insertHandler(err, insertErrors, apiResponse) {
  *   // err (object):
- *   //   An API error occurred.
- *
- *   // insertErrors (object[]):
- *   //   If populated, some rows failed to insert, while others may have
- *   //   succeeded.
- *   //
- *   // insertErrors[].row (original individual row object passed to `insert`)
- *   // insertErrors[].errors[].reason
- *   // insertErrors[].errors[].message
- *
- *   // See https://developers.google.com/bigquery/troubleshooting-errors for
- *   // recommendations on handling errors.
+ *   //   An API error or partial failure occurred.
+ *
+ *   if (err.name === 'PartialFailureError') {
+ *     // Some rows failed to insert, while others may have succeeded.
+ *
+ *     // err.errors (object[]):
+ *     // err.errors[].row (original row object passed to `insert`)
+ *     // err.errors[].errors[].reason
```

Comment:
Yeah :(

`err.errors` is the standard format from PartialFailureError. What goes in that array is up to the implementor; so in this case, a row can fail to insert for multiple reasons (according to [this doc](https://cloud.google.com/bigquery/docs/reference/rest/v2/tabledata/insertAll))